cibuildwheel/__main__.py

```diff
@@ -206,8 +192,7 @@ def _compute_platform_ci() -> PlatformName:
         return "windows"
     else:
         print(
-            'cibuildwheel: Unable to detect platform from "sys.platform" in a CI environment. You can run '
-            "cibuildwheel using the --platform argument. Check --help output for more information.",
+            'cibuildwheel: Unable to detect platform from "sys.platform". cibuildwheel doesn\'t support building wheels for this platform. You might be able to build for a different platform using the --platform argument. Check --help output for more information.',
```

Comment:
nitpick: I liked the variant with implicit string concat better since it's easier to consume shorter lines in code/diffs..

Reply:
updated